server/src/main/java/org/elasticsearch/cluster/coordination/JoinHelper.java

```diff
@@ -205,7 +205,7 @@ static Level getLogLevel(TransportException e) {
         }
 
         void logWarnWithTimestamp() {
-            logger.info(() -> new ParameterizedMessage("last failed join attempt was {} ago, failed to join {} with {}",
+            logger.warn(() -> new ParameterizedMessage("last failed join attempt was {} ago, failed to join {} with {}",
```

Comment:
D'oh this wasn't supposed to be here, it leaked from another branch. It's a good change IMO but completely unrelated. Just raising awareness in case there's any objections.